Scripts/Models (Under Development)/Nback/stim/Kane stimuli.py

```diff
@@ -0,0 +1,50 @@
+"""
```

Comment:
this can use raw string (starts with `r"""` instead of just `"""`) to avoid Python parsing the string.

Reply:
For the docstring?  (that's the only triple quotes in that file).  If so, why here an not everywhere?

Reply:
It can be used everywhere for docstrings, but changing the existing ones is a bit more involved effort. Python does not parse raw-strings, so the existing Python escapes (`\\`, `\n`, ...) need to be removed.
This is a new docstring so using raw string would get rid of the 'invalid decimal numeral' defect report and probably few 'invalid space sequence' warnings when the file is run.

Reply:
Right, the invalid decimal is probably for the variables, not sure why I see it as coming from the doc string. the part about raw-strings making docs cleaner should still apply.

Reply:
Got it.  Changed it to r""".  Thanks.